### PR TITLE
Cite reminders

### LINKED ISSUE #2597: Track updating the citation reminder strings
**Summary**

In order to complete the refactoring of the citation reminder we need to update all citation reminders to include a brief 1-line reference in the first line of the added string. By preference a DOI or some URL. This issue serves to keep track of the progress since it lists all files which currently have a citation reminder included.

Additional files may need to have citation reminders added (based on their documentation). Please see https://docs.lammps.org/Modify_style.html#citation-reminder-suggested for recommendations on when to add a reminder (and how many and when not).

**LAMMPS Version and Platform**

Starting with LAMMPS  10 Feb 2021

**Details**

- [x] compute_centroid_stress_atom.cpp
- [x] neighbor.cpp
- [ ] ASPHERE/pair_gayberne.cpp
- [ ] BOCS/fix_bocs.cpp
- [ ] COLVARS/fix_colvars.cpp
- [ ] DIELECTRIC/atom_vec_dielectric.cpp
- [x] DIFFRACTION/compute_saed.cpp
- [x] DIFFRACTION/compute_xrd.cpp
- [ ] DPD-MESO/pair_edpd.cpp
- [ ] DPD-MESO/pair_mdpd.cpp
- [ ] DPD-MESO/pair_tdpd.cpp
- [ ] DPD-REACT/fix_shardlow.cpp
- [ ] DPD-REACT/pair_multi_lucy.cpp
- [ ] DPD-REACT/pair_multi_lucy_rx.cpp
- [ ] EFF/atom_vec_electron.cpp
- [ ] EXTRA-COMPUTE/compute_stress_cartesian.cpp
- [ ] EXTRA-COMPUTE/compute_stress_cylinder.cpp
- [ ] EXTRA-COMPUTE/compute_stress_spherical.cpp
- [ ] EXTRA-FIX/fix_ave_correlate_long.cpp
- [ ] EXTRA-FIX/fix_electron_stopping_fit.cpp
- [ ] EXTRA-FIX/fix_filter_corotate.cpp
- [ ] EXTRA-FIX/fix_flow_gauss.cpp
- [ ] EXTRA-FIX/fix_pafi.cpp
- [ ] EXTRA-FIX/fix_rhok.cpp
- [ ] EXTRA-FIX/fix_ti_spring.cpp
- [ ] EXTRA-FIX/fix_ttm_mod.cpp
- [ ] EXTRA-MOLECULE/dihedral_table_cut.cpp
- [ ] EXTRA-PAIR/pair_e3b.cpp
- [ ] EXTRA-PAIR/pair_lj_relres.cpp
- [ ] EXTRA-PAIR/pair_momb.cpp
- [ ] GPU/fix_gpu.cpp
- [ ] INTERLAYER/pair_ilp_graphene_hbn.cpp
- [ ] INTERLAYER/pair_ilp_tmd.cpp
- [ ] INTERLAYER/pair_kolmogorov_crespi_full.cpp
- [ ] INTERLAYER/pair_saip_metal.cpp
- [x] KIM/kim_command.cpp
- [x] KIM/kim_init.cpp
- [x] KIM/pair_kim.cpp
- [ ] LATBOLTZ/fix_lb_fluid.cpp
- [ ] MANIFOLD/fix_nve_manifold_rattle.cpp
- [ ] MANIFOLD/fix_nvt_manifold_rattle.cpp
- [ ] MANYBODY/pair_atm.cpp
- [ ] MANYBODY/pair_edip_multi.cpp
- [ ] MANYBODY/pair_local_density.cpp
- [ ] MC/fix_bond_swap.cpp
- [ ] MC/fix_charge_regulation.cpp
- [ ] MISC/pair_agni.cpp
- [ ] MISC/pair_srp.cpp
- [ ] MISC/pair_srp_react.cpp
- [ ] ML-HDNNP/pair_hdnnp.cpp
- [ ] ML-RANN/pair_rann.cpp
- [ ] OPENMP/pair_reaxff_omp.cpp
- [ ] OPT/pair_ilp_graphene_hbn_opt.cpp
- [ ] ORIENT/fix_orient_bcc.cpp
- [ ] ORIENT/fix_orient_eco.cpp
- [ ] ORIENT/fix_orient_fcc.cpp
- [ ] PERI/atom_vec_peri.cpp
- [ ] PHONON/fix_phonon.cpp
- [ ] POEMS/fix_poems.cpp
- [ ] PTM/compute_ptm_atom.cpp
- [ ] REACTION/fix_bond_react.cpp
- [ ] REAXFF/fix_acks2_reaxff.cpp
- [ ] REAXFF/fix_qeq_reaxff.cpp
- [ ] REAXFF/pair_reaxff.cpp
- [ ] SPIN/fix_nve_spin.cpp
- [ ] SPIN/min_spin_cg.cpp
- [ ] SPIN/min_spin_lbfgs.cpp
- [ ] SPIN/neb_spin.cpp
- [ ] SRD/fix_srd.cpp
- [ ] UEF/fix_nh_uef.cpp

### PR DESCRIPTION
**Pull request by hammondkd**
**Summary**

Edits to BibTeX entries; catching all bug fixes in those entries that I found, whether they were in the list of ones that needed updates or not.

**Related Issue(s)**

Closes #2597 (except COLVARS)

**Author(s)**

Karl Hammond
University of Missouri
hammondkd at missouri.edu

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

This should not break or add to anything, it should only address issue #2597

**Implementation Notes**

I used the first one I found to determine what was desired for situations involving two or more citations. I'll fix them again if that wasn't what was needed.

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [ ] Licensing information is complete
- [ ] Corresponding author information is complete
- [ ] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

### CODE REVIEW

File: src/UEF/fix_nh_uef.cpp [hunk 1]
```diff
@@ -44,14 +44,16 @@ enum{ISO,ANISO,TRICLINIC};
 // citation info
 
 static const char cite_user_uef_package[] =
-  "UEF package:\n\n"
+  "UEF package: src:10.1063/1.4972894\n\n"
```
Comment (akohlmey) [inline src/UEF/fix_nh_uef.cpp]:
```suggestion
  "UEF package: doi:10.1063/1.4972894\n\n"
```

File: src/UEF/fix_nh_uef.cpp [hunk 2]
```diff
@@ -44,14 +44,16 @@ enum{ISO,ANISO,TRICLINIC};
...
   "@Article{NicholsonRutledge16,\n"
   "author = {David A. Nicholson and Gregory C. Rutledge},\n"
-  "title = {Molecular simulation of flow-enhanced nucleation in n-eicosane melts under steady shear and uniaxial extension},\n"
+  "title = {Molecular Simulation of Flow-Enhanced Nucleation in\n",
```
Comment (akohlmey) [inline src/UEF/fix_nh_uef.cpp]:
```suggestion
  "title = {Molecular Simulation of Flow-Enhanced Nucleation in"
```

Comment (hammondkd) [reply src/UEF/fix_nh_uef.cpp]: Fixed both of these

File: src/GPU/fix_gpu.cpp [hunk 3]
```diff
@@ -53,51 +53,57 @@ extern double lmp_gpu_update_bin_size(const double subx, const double suby, cons
...
+  " doi =     {10.1016/j.commatsci.2014.10.068},\n"
   " volume =  100,\n"
   " pages =   {173--180}\n"
   "}\n\n"
   "@Article{Trung17,\n"
   " author = {T. D. Nguyen},\n"
-  " title = {GPU-accelerated Tersoff potentials for massively parallel Molecular Dynamics simulations},\n"
-  " journal = {Comp.~Phys.~Comm.},\n"
+  " title = {{GPU}-Accelerated {T}ersoff Potentials for Massively Parallel Molecular Dynamics Simulations},\n"
+  " journal = {Comput.\ Phys.\ Commun.},\n"
```
Comment (akohlmey) [inline src/GPU/fix_gpu.cpp]:
```suggestion
  " journal = {Comput.\\ Phys.\\ Commun.},\n"
```

File: src/UEF/fix_nh_uef.cpp [hunk 4]
```diff
@@ -44,14 +44,16 @@ enum{ISO,ANISO,TRICLINIC};
...
   "@Article{NicholsonRutledge16,\n"
   "author = {David A. Nicholson and Gregory C. Rutledge},\n"
-  "title = {Molecular simulation of flow-enhanced nucleation in n-eicosane melts under steady shear and uniaxial extension},\n"
+  "title = {Molecular Simulation of Flow-Enhanced Nucleation in\n"
+  "   {$n$}-Eicosane Melts Under Steady Shear and Uniaxial Extension},\n"
   "journal = {The Journal of Chemical Physics},\n"
   "volume = {145},\n"
   "number = {24},\n"
   "pages = {244903},\n"
+  "doi = {10.1063/1.4972894},\n",
```
Comment (akohlmey) [inline src/UEF/fix_nh_uef.cpp]:
```suggestion
  "doi = {10.1063/1.4972894},\n"
```

File: src/UEF/fix_nh_uef.cpp [hunk 5]
```diff
@@ -44,14 +44,16 @@ enum{ISO,ANISO,TRICLINIC};
 // citation info
 
 static const char cite_user_uef_package[] =
-  "UEF package:\n\n"
+  "UEF package: doi:10.1063/1.4972894\n\n"
   "@Article{NicholsonRutledge16,\n"
   "author = {David A. Nicholson and Gregory C. Rutledge},\n"
-  "title = {Molecular simulation of flow-enhanced nucleation in n-eicosane melts under steady shear and uniaxial extension},\n"
+  "title = {Molecular Simulation of Flow-Enhanced Nucleation in\n"
```
Comment (akohlmey) [inline src/UEF/fix_nh_uef.cpp:50]:
```suggestion
  "title = {Molecular Simulation of Flow-Enhanced Nucleation in"
```

**Comment by akohlmey** [comment]
While looking for warnings about incorrect escapes, there also is:
``` 
" title     = {Carbon Tri-Interstitial Defect: {A} Model for {D$_{\mathrm{II}}$} Center},\n"
```
which should be:
```
 " title     = {Carbon Tri-Interstitial Defect: {A} Model for {D$_{\\mathrm{II}}$} Center},\n"
```
in file `src/MANYBODY/pair_edip_multi.cpp`

Also, please go to the "src" folder and type: `make fix-whitespace` to avoid whitespace warnings in the programming style check.

File: src/UEF/fix_nh_uef.cpp (hunk 4 above)
Comment (hammondkd) [reply src/UEF/fix_nh_uef.cpp]: This has been fixed and pushed

File: src/GPU/fix_gpu.cpp (hunk 3 above)
Comment (hammondkd) [reply src/GPU/fix_gpu.cpp]: Fixed this and some other BibTeX issues in the same file

**Comment by hammondkd** [comment]
Sorry this was a bit of a mess, but I think we may have finally gotten this one done. Axel, thanks for your help in tracking down typos.

**Review by sjplimp — APPROVED** [review]

**Pull request merged**